docs/essentials/datetime/PerformingArithmeticOperations.md

```diff
@@ -1 +1,186 @@
+
+```csharp
+using System;
+
+public enum TimeComparison
+{
+   EarlierThan = -1,
+   TheSameAs = 0,
+   LaterThan = 1
+}
+
+public class DateTimeOffsetManipulation
+{
+   public static void Main()
+   {
+      DateTimeOffset localTime = DateTimeOffset.Now;
+      DateTimeOffset utcTime = DateTimeOffset.UtcNow;
+
+      Console.WriteLine("Difference between local time and UTC: {0}:{1:D2} hours", 
+                        (localTime - utcTime).Hours, 
+                        (localTime - utcTime).Minutes);
+      Console.WriteLine("The local time is {0} UTC.", 
+                        Enum.GetName(typeof(TimeComparison), localTime.CompareTo(utcTime)));  
+   }
+}
+// Regardless of the local time zone, the example displays 
+// the following output to the console:
+//    Difference between local time and UTC: 0:00 hours.
+//    The local time is TheSameAs UTC.
+```
+
+In this example, the [DateTimeOffset.CompareTo](http://dotnet.github.io/api/System.DateTimeOffset.html#System_DateTimeOffset_CompareTo_System_DateTimeOffset_) method indicates that the current local time and the current UTC time are equal, and subtraction of `DateTimeOffset` values indicates that the difference between the two times is [TimeSpan.Zero](http://dotnet.github.io/api/System.TimeSpan.html#System_TimeSpan_Zero). 
+
+The chief limitation of using `DateTimeOffset` values in date and time arithmetic is that although `DateTimeOffset` values have some time zone awareness, they are not fully time zone aware. Although the `DateTimeOffset` value's offset reflects a time zone's offset from UTC when a `DateTimeOffset` variable is first assigned a value, it becomes disassociated from the time zone thereafter. Because it is no longer directly associated with an identifiable time, the addition and subtraction of date and time intervals does not consider a time zone's adjustment rules. 
+
+To illustrate, the transition to daylight saving time in the U.S. Central Standard Time zone occurs at 2:00 A.M. on March 9, 2008. This means that adding a two and a half hour interval to a Central Standard time of 1:30 A.M. on March 9, 2008, should produce a date and time of 5:00 A.M. on March 9, 2008. However, as the following example shows, the result of the addition is 4:00 A.M. on March 9, 2008. Note that this result of this operation does represent the correct point in time, although it is not the time in the time zone in which we are interested (that is, it does not have the expected time zone offset).
```

Comment:
We should use a combination of international time zones in our docs, not just US and not just from english speaking countries.